compiler/circle-inspect/src/Dump.cpp

```diff
@@ -133,3 +133,38 @@ void DumpConv2DWeight::run(std::ostream &os, const circle::Model *model)
 }
 
 } // namespace circleinspect
+
+namespace circleinspect
+{
+
+void DumpOperatorVersion::run(std::ostream &os, const circle::Model *model)
+{
+  std::map<std::string, int32_t> op_version_map;
+
+  circleinspect::Reader reader(model);
+
+  assert(reader.num_subgraph() == 1);
+  reader.select_subgraph(0);
```

Comment:
What about multiple graph models?

Reply:
I will handle it later because `DumpConv2DWeight` like functions also have same asserts :)

Reply:
OK, I think circle-inspect and tfl-inspect needs revise :)

Reply:
I've posted #3278